notebooks/240-EfficientDet-object-detection/README.md

```diff
@@ -0,0 +1,23 @@
+# EfficientDet Optimization with OpenVINO
+Research Paper : [EfficientDet: Scalable and Efficient Object Detection](https://openaccess.thecvf.com/content_CVPR_2020/papers/Tan_EfficientDet_Scalable_and_Efficient_Object_Detection_CVPR_2020_paper.pdf)
+<a href='https://github.com/google/automl/tree/master/efficientdet'>EfficientDets</a> are a family of object detection models, which achieve state-of-the-art 55.1mAP on COCO test-dev, yet being 4x - 9x smaller and using 13x - 42x fewer FLOPs than previous detectors. Our models also run 2x - 4x faster on GPU, and 5x - 11x faster on CPU than other detectors
```

Comment:
HTML tags should be replaced by Markdown. The paragraph cannot state "our model" as this is not ours. It should be rewritten to describe EfiicientDet as an external model.